.github/workflows/ruby.yml

```
@@ -0,0 +1,25 @@
+---
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+      - 'KATELLO-*'
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+
+  test:
+    name: Ruby
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@custom-db
```

https://github.com/theforeman/actions/commit/e4e8fb7c0e9907b6e0859306c4efd4240b763109 was merged
```suggestion
    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
```